lib/codelog/message.rb

```diff
@@ -32,6 +32,10 @@ def missing_config_file
           "Run the following command to create the missing file:\n\n" \
           'codelog setup'
         end
+
+        def could_not_parse_yaml(error)
+          "#{prefix}: #{error.problem.capitalize} in the line #{error.line} in #{error.file}"
```

Comment:
Perhaps the format 
```ruby
"#{prefix}: #{error.problem.capitalize} in #{error.file}:#{error.line}"
```

That way, the file name is easily copy-pastable, and will lead directly to the failing line